openlibrary/plugins/worksearch/subjects.py

```diff
@@ -240,7 +255,10 @@ def create_engine():
     sort_order = sort_options.get(sort) or sort_options['editions']
 
     engine = create_engine()
-    return engine.get_subject(key, details=details, offset=offset, sort=sort_order, limit=limit, **filters)
+    subject = engine.get_subject(
```

Comment:
Perhaps if this `subject` was named `result` like it is in `SubjectEngine.get_subject()`'s return, so subject is something like `cats`, and `result`  is a list of works with that subject? I found this confusing at first while reviewing (See above strikethrough comment)